interpreter/main.go

```diff
@@ -55,7 +56,9 @@ func saveHistory() {
 
 func executeLine(tx *sql.Tx, txnLine string) error {
 	if tidb.IsQuery(txnLine) {
+		start := time.Now()
```

Comment:
My rationale was to make sure we only report the elapsed time of query execution, therefore the timing was put as close as possible to the query execution instructions; actually I do agree with you refactoring `start := time.Now()` is cleaner.